src/ruby/yast/i18n.rb

```diff
@@ -60,7 +60,7 @@ def _(str)
           key_exist?(str)
         end
       end
-      found ? Translation._(str) : str
+      found ? Translation._(str) : str.freeze
```

Comment:
for consistency I would like to see freeze also on line 51

Reply:
and probably also do it for other methods like `n_`

In general I do not mind dup or without dup. Just please document it.